docusaurus/docs/learn/core-concepts/security/authentication/auth.md

```diff
@@ -211,21 +211,21 @@ A client may terminate its own API Session at any time by calling: `DELETE /edge
 
 ### Primary Authentication
 
-Primary authentication in Ziti establishes and API Sessions identity principal and enabled Ziti to determine which
+Primary authentication in Ziti establishes an API Session's identity principal and enables Ziti to determine which
 secondary authentication factors are necessary for an API Session to become fully authenticated. If no secondary
-authentication factors are required the API Session becomes fully authenticated immediately without any further
+authentication factors are required, the API Session becomes fully authenticated immediately without any further
 interaction with the Client or Management API.
 
 Primary authentication factors include:
 
-- x509 certificates
-- JWTs
-- Username/password
+- x509 certificates from primary Edge signer CA or trusted external CA
```

Comment:
I find the previous bullets better, including the capitalized P. I'd prefer the older text here.